### ISSUE TITLE
Документация

### ISSUE THREAD

**Issue by mes01** [MODX-Club/shopModx1C#1]
Документации не будет?

**Comment by Fi1osof** [comment]
@mes01 Нет. Этот компонент вообще давно уже не актуален. Его заменил https://github.com/MODX-Club/modImporter Про него я несколько подробных статей писал на modx.pro
И по нему тоже не будет документации.
